packages/@divvi/mobile/__mocks__/src/swap/UnfavorableRateBottomSheet.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
this was the easiest way to mock the slide button for SwapScreenV2 tests. Was having trouble with mocking directly in file. Open to other suggestions

Reply:
What do you think of moving `SlideButton` to its own file so it's easier to mock as needed?

Reply:
I don't think it makes much difference in terms of mocking for this PR. I think we need to do a similar mock file for the `SlideButton` component and we'd just have more files to maintain. Holding off on that for now until there's a second use for the slide button.